### PR TITLE
Add FasterTokenizer performance test

### PR DESCRIPTION
**Pull request by joey12300**
### PR types

Others
### PR changes

Others
### Description

add tf_tokenizer performance test;
use throughput instead of time to evaluate performance

### CODE REVIEW

File: examples/experimental/faster_tokenizer/demo.py [hunk 1]
```diff
@@ -2,6 +2,9 @@
 import os
```
Comment (ZeyuChen) [inline examples/experimental/faster_tokenizer/demo.py]: 文件名改为perf.py

File: examples/experimental/faster_tokenizer/demo.py [hunk 2]
```diff
@@ -73,9 +79,34 @@
...
 start = time.time()
-for _ in range(10):
+for _ in range(epochs):
     for batch_data in batches:
         encoded_inputs = hf_tokenizer(
             batch_data)  #, padding=True, truncation=True)
 end = time.time()
-print("hf_py_tokenizer: %.5f" % (end - start))
+print("hf_py_tokenizer: %.5f tokens/s" % (total_tokens / (end - start)))
+
```
Comment (ZeyuChen) [inline examples/experimental/faster_tokenizer/demo.py:107]: 补充下注释，用注释切分下
`# BERT Tokenizer using TensorFlow Text `

Comment (ZeyuChen) [reply examples/experimental/faster_tokenizer/demo.py:107]: 其他地方同理

File: examples/experimental/faster_tokenizer/demo.py [hunk 3]
```diff
@@ -2,6 +2,9 @@
 import os
 import time
 
+import tensorflow as tf
+import tensorflow_text as tf_text
+
```
Comment (ZeyuChen) [inline examples/experimental/faster_tokenizer/demo.py:19]: 清理一下一些无用的import，如get_path_from_url

File: examples/experimental/faster_tokenizer/demo.py (hunk 1 above)
Comment (joey12300) [reply examples/experimental/faster_tokenizer/demo.py]: done

File: examples/experimental/faster_tokenizer/demo.py (hunk 3 above)
Comment (joey12300) [reply examples/experimental/faster_tokenizer/demo.py:19]: done

File: examples/experimental/faster_tokenizer/demo.py (hunk 2 above)
Comment (joey12300) [reply examples/experimental/faster_tokenizer/demo.py:107]: done

**Review by ZeyuChen — APPROVED** [review]
LGTM

**Pull request merged**